dataland-frontend/src/components/forms/RetrieveSkyminder.vue

```diff
@@ -36,21 +36,20 @@
 
 <script>
 import {FormKit} from "@formkit/vue";
-import {SkyminderControllerApi} from "@/../build/clients/backend";
-import {DataStore} from "@/services/DataStore";
+import {SkyminderControllerApi} from "@/../build/clients/backend/api";
+import {ApiWrapper} from "@/services/ApiWrapper"
 import Card from 'primevue/card';
 import Button from 'primevue/button';
 import SkyminderTable from "@/components/ui/SkyminderTable";
 
-const api = new SkyminderControllerApi()
-const dataStore = new DataStore(api.getDataSkyminderRequest)
+const skyminderControllerApi = new SkyminderControllerApi()
+const getDataSkyminderRequestWrapper = new ApiWrapper(skyminderControllerApi.getDataSkyminderRequest)
```

Comment:
Aus Interesse würde ich grundsätzlich gerne verstehen, was es mit dem "Ersatz" des dataStores durch die Wrapper auf sich hat in diesem PR.